src/tools/tidy/src/features.rs

```diff
@@ -7,13 +7,22 @@
 //! * Library features have at most one stability level.
 //! * Library features have at most one `since` value.
 //! * All unstable lang features have tests to ensure they are actually unstable.
+//! * Language features in a group are sorted by `since` value.
 
 use std::collections::HashMap;
 use std::fmt;
 use std::fs::{self, File};
 use std::io::prelude::*;
 use std::path::Path;
 
+use regex::{Regex, escape};
+
+mod version;
+use self::version::Version;
```

Comment:
```
error[E0432]: unresolved import `version`
  --> src/tools/tidy/src/features.rs:21:5
   |
21 | use version::Version;
   |     ^^^^^^^ help: a similar path exists: `self::version`
```

Reply:
Oh... it's good you used `self::` here because it made me find out that I forgot to migrate the `tidy` crate to Rust 2018 in which `use version::Version;` would work fine.

Let's keep `use self::version::Version;` for now then. :)